### PR TITLE
Labels fill list from netdata configuration

### PR DESCRIPTION
**Pull request by thiagoftsm**
##### Summary
Finish #7410 

This PR brings the last bullets that were missed o the issue 7410.
##### Component Name
Daemon
Database

### CODE REVIEW

File: database/rrdhost.c [hunk 1]
```diff
@@ -712,10 +712,16 @@ static int is_valid_label_key(char *key) {
...
+    //Netdata internal
     if (*key == '_')
         return 0;
 
+    key++;
+    //Prometheus internal
+    if (*key == '_')
+        return 0;
+
+    key++;
```
Comment (ilyam8) [inline database/rrdhost.c]: i think there is no reason to check for `__` if `_` is already wrong (return 0). And i think you dont need `key++` after the check.

This is perfect case to use unit tests btw

Comment (thiagoftsm) [reply database/rrdhost.c]: I removed with the last commit, you right, I had a test that did not reach the loop when there was one unique `_` , so the second is not necessary.
I agree this is a perfect place for unique test.

File: database/rrdhost.c [hunk 2]
```diff
@@ -712,12 +712,7 @@ static int is_valid_label_key(char *key) {
...
 
-    //Netdata internal
-    if (*key == '_')
-        return 0;
-
-    key++;
-    //Prometheus internal
+    //Netdata and Prometheus  internal
     if (*key == '_')
         return 0;
```
Comment (ilyam8) [inline database/rrdhost.c]: `key++;` after is not needed too (befre `while` loop)

**Comment by thiagoftsm** [comment]
> The new labels are not loaded from the config file when the "reload-labels" command is used from the clitool.

I missed a call of a function, I removed the call from health_reload, but I did not put inside `load_config_labels()`, now it is ok.

Thank you!

**Pull request merged**